### PR TITLE
Mission completed! Search forms was created

### PR DESCRIPTION
**Pull request by RatuhWD**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,16 +1,98 @@
...
-    <h1>Search bar airbnb</h1>
+    <div class="container--l">
+      <form
+        class="form"
+        method="get"
+        data-qa="big"
+        action="/"
+      >
+        <label class="big-search__button button-general" for="bigSearchInput">
+          <svg
```
Comment (vlad-demchuk) [inline src/index.html:34]: it will be clearer to use a separate file for svg.

File: src/index.html [hunk 2]
```diff
@@ -1,16 +1,98 @@
...
+          >
+            <path
+              d="M13.8401 13.0856L13.5339 13.4375L13.8638 13.7673L18.4821 18.3846C18.4925 18.3949 18.5 18.4106 18.5 18.4313C18.5 18.4519 18.4925 18.4676 18.4821 18.4779L18.4821 18.4778L18.4755 18.4846C18.4677 18.4928 18.4538 18.5 18.4354 18.5C18.4318 18.5 18.4242 18.4991 18.4143 18.4951C18.4047 18.4911 18.3958 18.4852 18.3886 18.4779L13.7702 13.8606L13.4405 13.5309L13.0886 13.8369C11.7516 14.9996 10.0088 15.7027 8.10313 15.7027C3.91254 15.7027 0.5 12.2907 0.5 8.10133C0.5 3.91178 3.9085 0.5 8.10313 0.5C12.2939 0.5 15.7063 3.9079 15.7063 8.10133C15.7063 10.0065 15.0031 11.7489 13.8401 13.0856ZM8.09891 0.637472C3.98431 0.637472 0.633511 3.98276 0.633511 8.10133C0.633511 12.2195 3.98391 15.5694 8.09891 15.5694C12.2185 15.5694 15.5643 12.2149 15.5643 8.10133C15.5643 3.98736 12.2181 0.637472 8.09891 0.637472Z"
+              fill="#687C94"
+              stroke="#687C94"
+            >
+          </svg>
+        </label>
+
+
```
Comment (vlad-demchuk) [inline src/index.html:49]: one blank will be enough

File: src/reset.css [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
```
Comment (vlad-demchuk) [inline src/reset.css:85]: let's reset tags that you use for now instead of this long tag list

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  display: flex;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input-general {
+  background: none;
```
Comment (vlad-demchuk) [inline src/style.css:38]: why do you need `background: none`?

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
add a demo link

File: src/style.css (hunk 4 above)
Comment (RatuhWD) [reply src/style.css:38]: +

File: src/index.html (hunk 1 above)
Comment (RatuhWD) [reply src/index.html]: +

File: src/index.html (hunk 2 above)
Comment (RatuhWD) [reply src/index.html]: +

File: src/reset.css (hunk 3 above)
Comment (RatuhWD) [reply src/reset.css]: +

**Pull request closed without merging**